jina/serve/executors/decorators.py

```diff
@@ -129,6 +147,15 @@ def __set_name__(self, owner, name):
 
             setattr(owner, name, self.fn)
 
+        def _get_summary(
+            self, summary: Optional['Summary'], method_name: str
+        ) -> ContextManager:
+            return (
+                summary.labels(method_name).time()
```

Comment:
Can we also add the executor name as a label?

Reply:
Great idea, I will add it